README.md

```diff
@@ -86,8 +86,12 @@ Contributions, issues, and feature requests are welcome!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- KEY FEATURES -->
```

Comment:
- [optional] Kindly remove the template so that the sections will show up on your README file. 👍 `<!-- KEY FEATURES -->` Please change it to KEY FEATURES . Do the same for the future features as well.